website/app/templates/engineering.hbs

```diff
@@ -273,46 +288,126 @@
 </div>
 
 <div class="mt-8 prose prose-lg text-gray-800 pb-6">
-  <h2 class="ds-h2" id="use-other"><a href="#use-other" class="ds-a" rel="external">&sect;</a>
-    Alternative @hashicorp/flight-icons package</h2>
+  <h2 class="ds-h2" id="use-react"><a href="#use-react" class="ds-a" rel="external">&sect;</a>
+    Use in React apps</h2>
   <p>
     It is also possible to install
     <code class="ds-code">@hashicorp/flight-icons</code>
-    instead of
-    <code class="ds-code">@hashicorp/ember-flight-icons</code>. To do so, run:
+    and use the icons in React applications.
+  </p>
+  <p><em>
+      Notice: if you want to have more context you can
+      <a href="https://github.com/hashicorp/flight/pull/325" target="_blank" rel="noopener noreferrer">see the
+        pull-request here</a>
+      where this implementation has been discussed and agreed upon.</em>
+  </p>
+  <h3 class="ds-h3">Installation</h3>
+  <p>To install, run:
```

Comment:
```suggestion
  <p>To install, run:
  
     {{! prettier-ignore-start }}
    <CodeBlock @language="bash" @code="npm install @hashicorp/flight-icons" />
    {{! prettier-ignore-end }}
    
    or
```

non-blocking suggestion: we're kinda assuming everyone knows the difference between yarn/npm, saw this as confusing in the docs yesterday when pairing with React person

Reply:
agree, but it's an acceptable tradeoff vs make the documentation more complex for this detail

Reply:
makes sense, a-ok with me! maybe all the react ones can eventually become npm, but should verify if they do that most of the time. will keep an eye out